src/Microsoft.Azure.Batch.SoftwareEntitlement.Client.Native/README.md

```diff
@@ -86,10 +86,17 @@ if (err != 0)
 
 try
 {
+    //
+    // During development, it is useful to be able to validate against custom SSL
+    // certificates.  This logic would generally be removed in official releases.
```

Comment:
I'm not quite sure what this comment is telling me to do.  Are you saying they should remove the call to AddEntitlement for official releases?  Or they can add extra calls to AddEntitlement for dev releases?  Would it be appropriate to show a conditional approach, e.g.

```
#ifdef LOCAL_TEST
// use a sestest URL and validate against the custom cert
#else
// use the production URL from the Batch env var, and validate against the Microsoft certs
#endif
```